src/navigation/components/NavigationItem/components/GithubStars/GithubStars.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../../../../styles/variables';
+
+$block: '.#{$ns}github-stars';
+
+#{$block} {
+    display: flex;
+    align-items: center;
+
+    &.pc-header__button {
+        margin-top: 3px;
```

Comment:
I wrote this comment here. Doesn't change

https://github.com/gravity-ui/page-constructor/pull/159#discussion_r1113219849

Reply:
Sorry, i forgot to reply to your comment. We dont need reset-margin on this class because his dont change indents. However i wanted another solution without `pc-header__button`